src/formatPhoneNumber/index.js

```diff
@@ -0,0 +1,8 @@
+import * as R from "ramda";
+import throwError from "../throwError";
+
+export default (value) => {
+  !R.is(String, value) && throwError("value passed to formatPhoneNumber must be a string");
```

Comment:
Could we possibly allows both strings and numbers to be allowed? We could just convert a number phone into a string